apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -44,12 +48,16 @@ export class TeacherFeedback extends Component {
     this.onRubricChange = this.onRubricChange.bind(this);
 
     const {latestFeedback} = this.props;
-
     this.state = {
       comment: latestFeedback?.comment || '',
       performance: latestFeedback?.performance,
       studentId: studentId,
```

Comment:
I'll set `this.studentId` instead of having it set on the state as a first step. I'm not sure we actually store the user that the teacher is viewing in redux

Reply:
👍